internal/config/config.go

```diff
@@ -34,25 +34,31 @@ type Default struct {
 
 	// TZ represent system time location .
 	TZ string `json:"time_zone" yaml:"time_zone"`
+
+	// Log represent log configuration.
+	Log `json:"log" yaml:"log"`
 }
 
 func (c *Default) Bind() *Default {
 	c.Version = GetActualValue(c.Version)
 	c.TZ = GetActualValue(c.TZ)
+	c.Log = c.Log.Bind()
 	return c
 }
 
 func (c *Default) UnmarshalJSON(data []byte) (err error) {
 	ic := new(struct {
 		Ver string `json:"version"`
 		TZ  string `json:"time_zone"`
+		Log `json:"log"`
```

Comment:
same

Reply:
thank you for your review!
I will fix soon.